source/_integrations/synology_dsm.markdown

```diff
@@ -54,14 +54,14 @@ If you denied access to all locations and applications it is normal to receive a
 ## Sensors
 
 Utilisation:
-- `cpu_other_load`: Displays unspecified (that is, not user or system) load in percentage.
-- `cpu_user_load`: Displays user load in percentage.
-- `cpu_system_load`: Displays system load in percentage.
-- `cpu_total_load`: Displays combined load in percentage.
-- `cpu_1min_load`: Displays maximum load in past minute.
-- `cpu_5min_load`: Displays maximum load in past 5 minutes.
-- `cpu_15min_load`: Displays maximum load in past 15 minutes.
-- `memory_real_usage`: Displays percentage of memory used.
+- `cpu_load_other`: Displays unspecified (that is, not user or system) load in percentage.
+- `cpu_load_user`: Displays user load in percentage.
+- `cpu_load_system`: Displays system load in percentage.
+- `cpu_load_total`: Displays combined load in percentage.
+- `cpu_load_1min`: Displays maximum load in past minute.
+- `cpu_load_5min`: Displays maximum load in past 5 minutes.
+- `cpu_load_15min`: Displays maximum load in past 15 minutes.
+- `memory_usage_real`: Displays percentage of memory used.
 - `memory_size`: Displays total size of memory in MB.
 - `memory_cached`: Displays total size of cache in MB.
 - `memory_available_swap`: Displays total size of available swap in MB.
```

Comment:
This integration uses a configuration flow and the entity registry. The entity names are therefore never guaranteed. It is better to describe a list of provided sensors instead.